sys-utils/lscpu-cputype.c

```diff
@@ -192,12 +193,17 @@ static const struct cpuinfo_pattern type_patterns[] =
 {
 	/* Sort by fields name! */
 	DEF_PAT_CPUTYPE( "ASEs implemented",	PAT_FLAGS,	flags),		/* mips */
+	DEF_PAT_CPUTYPE( "Address Sizes",	PAT_ADDRESS_SIZES,	addrsz),/* loongarch */
```

Comment:
It's a bit sad that so many fields are not using the default spelling...

Reply:
Unfortunately the differences have not been highlighted as an issue during arch/loongarch review when we made the kernel port.  And changing the content of /proc/cpuinfo now will be considered a kernel public API break :(.

Reply:
Yeah, I understand. My comment is not actionable.